### ISSUE TITLE
Probably want bulk function compilation

### ISSUE THREAD

**Issue by workingjubilee** [tcdi/plrust#206]
Considering the `pg_dump` and `pg_restore` use case: By the time we get to large suites of thousands of functions, we're gonna want to have the ability to rebuild everything in one go.

**Comment by JohnHVancouver** [comment]
OOC what would be the use case? To attempt to compile everything with the latest set of lints? I think you could create a plpgsql wrapper around the information in pg_proc technically. There use to be a `recompile_function` in earlier versions that I think the intent was to be used by read replicas backed when things were stored on disk?

**Comment by workingjubilee** [comment]
Yes. Our current assumption is that the contents of the `pg_dump` someone uses `pg_restore` on should not be trusted, and may not match the current environment, nor configuration, nor etc., and we should rerun the compilation process from the top.
